tests/test_utils.py

```diff
@@ -5,15 +5,22 @@
 from __future__ import absolute_import, unicode_literals
 
 from contextlib import contextmanager
+from unittest import mock
 
+import ddt
 from django.contrib.auth.models import AnonymousUser, User
 from django.test import RequestFactory, TestCase
-from mock import patch
 
 from edx_rbac.utils import (
     ALL_ACCESS_CONTEXT,
+    _user_has_access,
+    contexts_accessible_from_request,
     create_role_auth_claim_for_user,
+    get_decoded_jwt,
+    has_access_to_all,
+    is_iterable,
     request_user_has_implicit_access_via_jwt,
+    set_from_collection_or_single_item,
     user_has_access_via_database
 )
```

Comment:
Nit: At this point I'd say let's import `utils` and then just do `utils.thing`, but I understand if you don't want to change that all now

Reply:
:disappears: